gossip/src/duplicate_shred_handler.rs

```diff
@@ -131,6 +137,24 @@ impl DuplicateShredHandler {
                     shred1.into_payload(),
                     shred2.into_payload(),
                 )?;
+                let should_notify = {
+                    let root_bank = self.bank_forks.read().unwrap().root_bank();
+                    let activated_slot = root_bank
+                        .feature_set
+                        .activated_slot(&feature_set::enable_gossip_duplicate_proof_ingestion::id())
+                        .map(|slot| root_bank.epoch_schedule().get_epoch(slot));
+                    activated_slot.is_some_and(|feature_epoch| {
+                        root_bank.epoch_schedule().get_epoch(slot) > feature_epoch
+                            // feature_epoch could only be 0 in tests and new cluster setup.
+                            || feature_epoch == 0
+                    })
+                };
```

Comment:
instead of a local variable, can we move this to a helper function or something?